### PR TITLE
Qae1162 collaborators fixes

### PR DESCRIPTION
**Pull request by RuslanHamidullin**
* Switched from using fancybox to simple redirects in views as we need this working with Javascript turned off too
* Added some missing Rspec tests

### CODE REVIEW

File: app/controllers/account/collaborators_controller.rb [hunk 1]
```diff
@@ -32,12 +32,22 @@ def create
       account, 
       create_params).run
     self.collaborator = add_collaborator_interactor.collaborator
+
+    if add_collaborator_interactor.success?
+      redirect_to account_collaborators_path, 
+                  notice: "#{collaborator.email} successfuly added to Collaborators!"
```
Comment (dreamfall) [inline app/controllers/account/collaborators_controller.rb]: typo: `successfully`. Same for notice below.

File: app/views/account/collaborators/new.html.slim [hunk 2]
```diff
@@ -0,0 +1,16 @@
+- provide(:page_content_class, "multi-page page-account")
+
+header.page-header.group class="#{'page-header-wider' unless current_user.completed_registration}"
```
Comment (dreamfall) [inline app/views/account/collaborators/new.html.slim]: Minor thing: no need to use string interpolation
```ruby
class=('page-header-wider' unless current_user.completed_registration)
```

File: app/controllers/account/collaborators_controller.rb (hunk 1 above)
Comment (RuslanHamidullin) [inline app/controllers/account/collaborators_controller.rb]: ahhha :bomb:

**Comment by RuslanHamidullin** [comment]
Gonna fix, thanks :dancers:

**Pull request merged**